sdk/storage/storage-file-datalake/src/generated/src/models/index.ts

```diff
@@ -1044,12 +1058,10 @@ export interface PathLeaseOptionalParams extends coreHttp.OperationOptions {
   requestId?: string;
   /** The timeout parameter is expressed in seconds. For more information, see <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/setting-timeouts-for-blob-service-operations">Setting Timeouts for Blob Service Operations.</a> */
   timeout?: number;
-  /** The lease duration is required to acquire a lease, and specifies the duration of the lease in seconds.  The lease duration must be between 15 and 60 seconds or -1 for infinite lease. */
-  xMsLeaseDuration?: number;
```

Comment:
Why our CI didn't find this regression?

Reply:
The datalake SDK doesn't have methods for leasing operation, so no interface would need to set this value.
 .Net SDK also doesn't have leasing operation methods. 

The change is because the Swagger spec added an option to put the parameter in client instance instead of method: : `"x-ms-parameter-location": "client"`. The SDK doesn't expose any interface, so it has no impact on customer.  But I'd still need to check with Sean why made this change.